tests/base/2d_2cyl_model

```diff
@@ -0,0 +1,48 @@
+135 47
```

Comment:
@thast: is this for testing, if you slack me the file, I will upload and send you a url where you can download it so it is not in the core repo

Reply:
@lheagy : Thanks for pointing it out. The test actually writes and then read back the testing files here (so we test both input and output). 
I will reproduce what we do in SimPEG and create a temporary folder that we then delete, so no storing will be necessary.